coordinator/ethereum/blob-submitter/src/main/kotlin/net/consensys/zkevm/ethereum/finalization/AggregationFinalizationCoordinator.kt

```diff
@@ -219,7 +220,12 @@ class AggregationFinalizationCoordinator(
         lineaRollup = lineaRollup,
         aggregationsRepository = aggregationsRepository,
         blobsRepository = blobsRepository,
-        aggregationSubmitter = AggregationSubmitterImpl(lineaRollup, gasPriceCapProvider),
+        aggregationSubmitter = AggregationSubmitterImpl(
```

Comment:
Would it be better if we inject `AggregationSubmitter` into `AggregationFinalizationCoordinator`? 

`aggregationSubmittedEventConsumer` isn't used by `AggregationFinalizationCoordinator` at all, it's just used to create `AggregationSubmitterImpl` and this creates a redundant dependency